website/pages/docs/agent/config-entries/proxy-defaults.mdx

```diff
@@ -19,22 +19,22 @@ one global entry is supported.
 Set the default protocol for all sidecar proxies in the default namespace:
 
 ```hcl
-kind = "proxy-defaults"
-name = "global"
-namespace = "default"
-config {
-  protocol = "http"
+Kind      = "proxy-defaults"
+Name      = "global"
+Namespace = "default"
+Config {
+  Protocol = "http"
```

Comment:
```suggestion
  protocol = "http"
```

Haven't tested this all the way through, but this might still need to be lowercase `protocol` because the `config` field is passed through as an arbitrary bag of data and the struct that uses it only declares `protocol` as the field name.

At the very least, this would maintain consistency with the example down below, where we keep `handshake_timeout_ms` as snake case in the `Config` block.

Reply:
Ok sounds good.